packages/drivers/local-driver/src/localCreateDocument.ts

```diff
@@ -20,8 +20,12 @@ export async function createDocument(
 ) {
 	const pathName = new URL(resolvedUrl.url).pathname;
 	const pathArr = pathName.split("/");
-	const tenantId = pathArr[pathArr.length - 2];
-	const id = pathArr[pathArr.length - 1];
+	// Why are we non null asserting here
```

Comment:
Why is this text here?

Reply:
Please see https://github.com/microsoft/FluidFramework/pull/21659#discussion_r1664495176